code/API_definitions/location-verification.yaml

```diff
@@ -80,25 +80,23 @@ info:
     # Further info and support
 
     (FAQs will be added in a later version of the documentation)
-  termsOfService: http://swagger.io/terms/
-  contact:
-    email: project-email@sample.com
+  version: 1.0.0-rc.1
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.3.0-wip
+  x-camara-commonalities: 0.4.0
 externalDocs:
   description: Product documentation at CAMARA
   url: https://github.com/camaraproject/
 servers:
-  - url: "{apiRoot}/location-verification/v0"
+  - url: "{apiRoot}/location-verification/v1"
```

Comment:
Should be - url: "{apiRoot}/location-verification/v1rc1" ?

@hdamker could you confirm?

Reply:
Yes, no doubt, I'll correct it